lyricsgenius/api.py

```diff
@@ -474,20 +473,29 @@ def _get_item_from_search_response(self, response, search_term, type_, result_ty
         """
 
         # Convert list to dictionary
-        hits = response['sections'][0]['hits']
+        top_hits = response['sections'][0]['hits']
 
         # Check rest of results if top hit wasn't the search type
         sections = sorted(response['sections'],
-                          key=lambda sect: sect['type'] == type_,
-                          reverse=True)
```

Comment:
This reverses the hits and brings the bottom result on top. Was there a reason `reverse` was set as True?

Reply:
The sorting was added with PR #91. The sorting was supposed to make it more likely that the user's intended search type is returned, but I can't remember why `reverse` was set to True. I'm fine with the change you've made. We'll see if anything breaks!